src/satellite_images_nso/_manipulation/nso_manipulator.py

```diff
@@ -309,13 +310,7 @@ def transform_vector_to_pixel_df(path_to_vector, add_ndvi_column=False):
     df = pd.DataFrame(data=data)
     geometry = gpd.points_from_xy(df.X, df.Y)
 
-    if add_ndvi_column == True:
-        df["ndvi"] = calculate_nvdi.normalized_diff(
-            src.read()[3][mask], src.read()[2][mask]
-        )
-        df = df[["blue", "green", "red", "nir", "ndvi", "X", "Y"]]
-    else:
-        df = df[["blue", "green", "red", "nir", "X", "Y"]]
+    df = df[["blue", "green", "red", "nir", "X", "Y"]]

```

Comment:
What happens if there are more/other channels? (NED, Deep Blue for instance)

Reply:
no pneo at the time of writing that code

Reply:
outdated function is deleted now because this is in the data loader now